### PR TITLE
Jd/updates lits04

### PR DESCRIPTION
**Pull request by jd-lara**
cc. @rodrigomha

### CODE REVIEW

File: src/utils/power_flow/make_pf.jl [hunk 1]
```diff
@@ -66,6 +71,10 @@ function make_pf(system)
         P_LOAD_BUS[ix], Q_LOAD_BUS[ix] = _get_load_data(system, b)
 
         if b.bustype == BusTypes.REF
+            injection_components_ = get_components(StaticInjection, system)
```
Comment (daniel-thom) [inline src/utils/power_flow/make_pf.jl:74]: You could use the new filter_func here.

File: src/utils/power_flow/power_flow.jl [hunk 2]
```diff
@@ -64,19 +64,19 @@ function _update_branch_flow!(sys::System)
...
+    result = round.(nl_result.zero; digits = 7)
     buses = enumerate(sort(collect(get_components(Bus, sys)), by = x -> get_number(x)))
 
     for (ix, bus) in buses
         if bus.bustype == BusTypes.REF
             P_gen = result[2 * ix - 1]
             Q_gen = result[2 * ix]
-            injection_components = get_components(Generator, sys)
-            devices = [d for d in injection_components if d.bus == bus]
+            injection = get_components(StaticInjection, sys)
```
Comment (daniel-thom) [inline src/utils/power_flow/power_flow.jl:74]: Same here.

File: src/models/dynamic_generator.jl [hunk 3]
```diff
@@ -15,11 +15,12 @@ mutable struct DynamicGenerator{
     machine::M
     shaft::S
     avr::A
-    tg::TG
+    prime_mover::TG
```
Comment (rodrigomha) [inline src/models/dynamic_generator.jl:18]: should we change to `prime_mover::PM` for consistency?

File: src/models/generated/Source.jl [hunk 4]
```diff
@@ -6,8 +6,8 @@ This file is auto-generated. Do not edit.
         name::String
         available::Bool
         bus::Bus
-        V_R::Float64
-        V_I::Float64
+        activepower::Float64
```
Comment (rodrigomha) [inline src/models/generated/Source.jl:9]: This change will require to properly set-up new equations in LITS for Source devices.

**Pull request merged**